main.py

```diff
@@ -1,18 +1,22 @@
 from src import (
     parse_input,
-    commands,
+    contact_commands,
     exit_commands,
+    help_commands,
     load_data,
     save_data,
     get_autocomplete_input,
 )
 
 
 def main():
-    FILENAME = "addressbook"
-    book = load_data(FILENAME)
-
-    all_commands = [c for c in commands] + exit_commands
+    ADRESBOOK_FILENAME = "addressbook"
+    book = load_data(ADRESBOOK_FILENAME)
+    all_commands = (
+        [c for c in help_commands]
+        + [c for c in contact_commands]
+        + [c for c in exit_commands]
+    )
```

Comment:
```suggestion
    all_commands = [c for c in (help_commands + contact_commands + exit_commands)]
```

Reply:
unfortunately, it doesn't work like that with dict